WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/ListHeaderView.swift

```diff
@@ -0,0 +1,54 @@
+import SwiftUI
+
+/// Renders a Section Header/Footer with a label [Left / Center/Right].
+///
+struct ListHeaderView: View {
+    @State var text: String
+    @State var alignment: TextAlignment
+
+    var body: some View {
+        Section() {
+            HStack() {
+                if alignment == .right {
+                    Spacer()
+                }
+                Text(text)
+                    .font(.footnote)
+                    .foregroundColor(Color(.listIcon))
+                if alignment == .left {
+                    Spacer()
+                }
+            }.frame(minWidth: 0, maxWidth: .infinity)
+        }
+        .padding([.leading, .trailing], 16)
+        .padding(.top, 8)
+        .frame(height: 42)
```

Comment:
Same question about having these magic numbers in enums. Just pointing them out in case we decide to change it, we don't miss them.